packages/mint-components/src/components/tax-and-cash/sqm-banking-info-form/sqm-banking-info-form-view.tsx

```diff
@@ -0,0 +1,355 @@
+import { h } from "@stencil/core";
+import { createStyleSheet } from "../../../styling/JSS";
+import { intl } from "../../../global/global";
+
+export interface BankingInfoFormViewProps {
+  states: {
+    loading: boolean;
+    disabled: boolean;
+    hideSteps: boolean;
+    hideBanking?: boolean;
+    hidePayPal?: boolean;
+    feeCap?: string;
+    isPartner: boolean;
+    formState: {
+      checked: "toBankAccount" | "toPaypalAccount" | undefined;
+      errors?: any;
+    };
+  };
+  slots: {
+    formInputsSlot?: any;
+    countryInputSlot?: any;
+    paymentMethodSlot?: any;
+  };
+  callbacks: {
+    setChecked: (
+      checked: "toBankAccount" | "toPaypalAccount" | undefined
+    ) => void;
+    onSubmit: (props: any) => Promise<void>;
+    onChange: (e) => void;
+  };
+  text: {
+    formStep: string;
+    taxAndPayouts: string;
+    taxAndPayoutsDescription: string;
+    directlyToBankAccount: string;
+    toPaypalAccount: string;
+    paymentMethod: string;
+    submitButton: string;
+    payPalInputLabel: string;
+    isPartnerAlertHeader: string;
+    isPartnerAlertDescription: string;
+    error: {};
```

Comment:
`{}` usually isn't the right type for an object, consider using `Object` or something more specific